### PR TITLE
Pc 603

### PR DESCRIPTION
**Pull request by matrix-drorgo**

### CODE REVIEW

File: src/Components/Generic/PopupComponents/PopUpFormTemplates/MainPopUpFormTemplate.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import Grid from '@material-ui/core/Grid';
+import React from 'react';
+import { connect } from 'react-redux';
+import { StyledRoundButton } from 'Components/Generic/PopupComponents/PopUpFormTemplates/Style';
+import { useTranslation } from 'react-i18next';
+import PopUpContantList from 'Components/Generic/PopupComponents/PopUpFormTemplates/PopUpContantList';
+import PopUpContext from 'Components/Generic/PopupComponents/PopUpFormTemplates/PopUpContext';
+import l from 'Assets/Images/l.png';
```
Comment (matrix-amiel) [inline src/Components/Generic/PopupComponents/PopUpFormTemplates/MainPopUpFormTemplate.js:8]: please replace the image name to meaningful name :)

Comment (matrix-drorgo) [reply src/Components/Generic/PopupComponents/PopUpFormTemplates/MainPopUpFormTemplate.js:8]: I don't understand this is meanigful this is the main of the two components below ?

File: src/Utils/Services/FHIR/CommonFunctions/index.js [hunk 2]
```diff
@@ -12,3 +12,15 @@ export const convertParamsToUrl = (obj) => {
   }
   return str;
 };
+
+export const arrayRemove = (arr, value) => {
```
[the remark this answers is not in the document]
Comment (matrix-drorgo) [reply src/Utils/Services/FHIR/CommonFunctions/index.js:16]: image

where there is no sevice/healpers

[the remark this answers is not in the document]
Comment (matrix-amiel) [reply src/Utils/Services/FHIR/CommonFunctions/index.js:16]: my mistake... Utils/helpers/CommonFunctions

File: src/Components/Generic/PopupComponents/PopUpFormTemplates/MainPopUpFormTemplate.js (hunk 1 above)
Comment (matrix-amiel) [reply src/Components/Generic/PopupComponents/PopUpFormTemplates/MainPopUpFormTemplate.js:8]: what is 1.png ?

Comment (matrix-drorgo) [reply src/Components/Generic/PopupComponents/PopUpFormTemplates/MainPopUpFormTemplate.js:8]: I don't understand where do you see 1.png as we talked it is l.png not 1.png and it comes from here 
image
I can change it to left.png but please for further development talk to the designer to call the names by the names you like so we don't need to change it but only to take it from the zeplin.

Comment (matrix-drorgo) [reply src/Components/Generic/PopupComponents/PopUpFormTemplates/MainPopUpFormTemplate.js]: FIXED...

File: src/Utils/Services/FHIR/CommonFunctions/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (matrix-drorgo) [reply src/Utils/Services/FHIR/CommonFunctions/index.js:16]: FIXED...

**Pull request merged**